iocage/lib/Jail.py

```diff
@@ -501,6 +520,25 @@ def _run_poststop_hook_manually(self) -> None:
         self.logger.debug("Running poststop hook manually")
         iocage.lib.helpers.exec(self.get_hook_script_path("poststop"))
 
+    def _wrap_jail_command(
+        self,
+        commands: typing.Optional[typing.List[str]]
+    ) -> typing.List[str]:
+        """Wrap a jail hook command for a host hook script."""
+        if commands is None:
+            return []
+
+        EOF_IDENTIFIER = f"EOF{random.getrandbits(64)}"
+        output: typing.List[str] = [
+            "set -e",
+            "echo 'Executing jail start scripts'",
+            "jexec -j {self.identifier} /bin/sh <<{EOF_IDENTIFIER}"
+        ] + commands + [
```

Comment:
does `+` automatically expand as `commands.join("\n")`?

Reply:
This concatenates multiple lists of string (such as `commands`) that are later on joined.

Python joins are the other way around: `"\n".join(commands)` by the way.